firebase-firestore/src/test/java/com/google/firebase/firestore/local/SQLiteSchemaTest.java

```diff
@@ -571,6 +571,35 @@ public void rewritesCanonicalIds() {
             });
   }
 
+  @Test
+  public void rewritesDocumentKeys() {
```

Comment:
I'd like to also have a test where the number of documents to migrate exceeds the batch size, to make sure batching works.

Reply:
Done